variables.tf

```diff
@@ -25,9 +25,21 @@ variable "dd_site" {
 }
 
 variable "integration_name" {
-  type = string
+  type        = string
   description = "Name of the integration to create."
-  default = "Datadog metrics"
+  default     = "Datadog"
+}
+
+variable "send_logs" {
+  type        = bool
+  description = "Whether to send logs to Datadog."
+  default     = true
```

Comment:
It might be safer to default this to `false` so that existing users of the module don't suddenly start sending through run logs without expecting it. Especially because DataDog's billing is affected by how much logs you index, so depending on how customers have DataDog setup it could be pricey.